src/components/v5/common/Modals/InviteMembersModal/InviteMemberModal.tsx

```diff
@@ -71,26 +88,52 @@ const InviteMembersModal = ({ isOpen, onClose }: Props) => {
       </div>
       <CardWithCallout
         title={
-          <span className="rounded-lg bg-gray-100 p-2 text-sm font-medium text-gray-900">
+          <span
+            className={clsx(
+              'rounded-lg bg-gray-100 p-2 text-sm font-medium text-gray-900',
+              {
+                'bg-negative-100 text-negative-400': isOutOfInvites,
+              },
+            )}
+          >
             <FormattedMessage
               {...MSG.invitesUsed}
               values={{ invitesAvailable, invitesUsed }}
             />
           </span>
         }
-        subtitle={<FormattedMessage {...MSG.inviteLinkHeading} />}
-        button={
-          <Button
-            text={MSG.buttonText}
-            mode={isCopied ? 'completed' : 'quinary'}
-            icon={isCopied ? undefined : CopySimple}
-            onClick={() => handleClipboardCopy(inviteLink)}
-            size="small"
-            textValues={{ isCopied }}
+        subtitle={
+          <FormattedMessage
+            {...(isOutOfInvites ? MSG.limitReached : MSG.inviteLinkHeading)}
           />
         }
+        button={
+          !isOutOfInvites ? (
+            <Button
+              text={MSG.buttonText}
+              mode={isCopied ? 'completed' : 'quinary'}
+              icon={isCopied ? undefined : CopySimple}
+              onClick={() => handleClipboardCopy(inviteLink)}
+              size="small"
+              textValues={{ isCopied }}
+            />
+          ) : (
+            <Link
+              to={`https://colony.io/request-member-invites?colony=${colonyName}`}
```

Comment:
Can this be added to the const file with all other URLs so that it can easily be updated.